### ISSUE TITLE
remove alpine from CITGM runs

### ISSUE THREAD

**Issue by mcollina** [nodejs/citgm#1069]
I recommend we skip alpine from CITGM runs, because it does not have the IPv6 network.

**Comment by targos** [comment]
Another option is to setup IPv6. Alpine runs in containers, maybe it's an easy change to the Dockerfile.

**Comment by targos** [comment]
I had a look and I think we would need to configure ipv6 at the Docker daemon level: https://docs.docker.com/config/daemon/ipv6/#use-ipv6-for-the-default-bridge-network

@nodejs/build-infra wdyt?

It seems relatively important to me that we keep running CITGM on Alpine hosts because many people use it in production.

**Comment by SimenB** [comment]
>It seems relatively important to me that we keep running CITGM on Alpine hosts because many people use it in production.

Agreed. Anecdotally, but I don't think I've encountered any node projects in production that does not use alpine (except for some that need specific native dependencies that are way easier with a debian distro)

**Comment by sxa** [comment]
Particularly with the move to get Alpine/x64 to tier 2 I'd say it's increasingly important to keep this running. It's a bit of a shame if citgm (I appreciate it's likely individual modules rather than citgm itself!) cannot pass if IPv6 is absent though.
